mudlet-mapper.xml

```diff
@@ -7730,11 +7730,32 @@ end</script>
   end
 end
 
-function mmp.roomFind(query)
+function mmp.roomFind(query, lines)
   if query:ends('.') then
     query = query:sub(1, -2)
   end
+  local defaultLine = 30 --could this to a setting
   local result = mmp.searchRoom(query)
+  if lines == 'all' then
+    lines = table.size(result)
+  end
+  lines = (lines ~= '') and tonumber(lines) or defaultLine 
+  
+  --create a new table (_nt) with keys and add areas to the table
+  local _nt = {}
```

Comment:
Could you give this a better name? We don't use the underscore for private variables either

Reply:
renamed and uploaded!